src/main/kotlin/crypto_service/service/EncryptionService.kt

```diff
@@ -28,19 +28,28 @@ class EncryptionService {
                     when (waitFor()) {
                         EXECUTION_STATUS_OK -> result
                         else -> throw SopsEncryptionException(
-                            message = "Failed when encrypting RiSc with ID: $riScId by running sops command: ${toEncryptionCommand(config, gcpAccessToken.sensor().value)} with error message: $result",
+                            message = "Failed when encrypting RiSc with ID: $riScId by running sops command: ${
+                                toEncryptionCommand(
+                                    config,
+                                    gcpAccessToken.sensor().value
+                                )
+                            } with error message: $result",
                             riScId = riScId
                         )
                     }
                 }
         } catch (e: Exception) {
+            print(e.message)
```

Comment:
`logger` 🤓 
I #7 har jeg lagt til en logger, du kan ta en titt der for hvordan man instansierer loggeren!